e2e/actions/l2_challenger_test.go

```diff
@@ -13,14 +15,16 @@ import (
 	"github.com/kroma-network/kroma/components/node/testlog"
 	chal "github.com/kroma-network/kroma/components/validator/challenge"
 	"github.com/kroma-network/kroma/e2e/e2eutils"
+	"github.com/kroma-network/kroma/e2e/testdata"
 )
 
-const lastValidBlockNum = uint64(6)
+const dummyHash = "0xaf01bc158f9b35867aea1517e84cf67eedc6a397c0df380b4b139eb570ddb2fc"
```

Comment:
Could you make them public and rename this to `DummyHashSepolia` or something like that?

Reply:
I see. I declared constants for them in `e2e/constants.go`.